example/src/App.web.tsx

```
@@ -1408,6 +1853,13 @@ const styles = StyleSheet.create({
     marginBottom: 20,
   },
   sectionTitle: { fontSize: 18, fontWeight: 'bold', marginBottom: 12 },
+  subheading: {
+    fontSize: 15,
+    fontWeight: '600',
+    marginTop: 16,
+    marginBottom: 8,
+  },
+  hint: { fontSize: 12, color: '#666', marginBottom: 8 },
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)